src/prajakta/Exam2.java

```diff
@@ -0,0 +1,40 @@
+package prajakta;
+
+public class Exam2 {
+
+	   int sum = 6;
+	   void findPairsOfSum(int[] arr) {
+		       for(int index = 0; index<arr.length; index++) {
+		    	       int num = arr[index];
+		    	       for(int innerIndex=0; innerIndex<arr.length; innerIndex++) {
+		    	    	       int num1=0;
+		    	    	       if(num<arr[innerIndex])
```

Comment:
I would recommend to check below code
https://github.com/KrishnaKTechnocredits/JAVATechnoDec2021/pull/850/files
https://github.com/KrishnaKTechnocredits/JAVATechnoDec2021/pull/845/files